packages/pkl.impl.ghactions/PklProject

```diff
@@ -17,11 +17,20 @@
 amends "../basePklProject.pkl"
 
 package {
-  version = "1.0.1"
+  version = "1.1.0"
 }
 
 dependencies {
   ["com.github.actions"] {
     uri = "package://pkg.pkl-lang.org/pkl-pantry/com.github.actions@1.1.0"
   }
+  ["deepToTyped"] {
+    uri = "package://pkg.pkl-lang.org/pkl-pantry/pkl.experimental.deepToTyped@1.1.1"
+  }
+}
+
+evaluatorSettings {
+  externalProperties {
+    ["mode"] = "test"
+  }
 }
```

Comment:
Should this really be set here?

Reply:
Any concerns? The intention is to help with testing when using the vanilla CLI, e.g. `cd packages/pkl.impl.ghactions && pkl test`

Reply:
Ahh okay that makes sense, no concerns.